src/Microsoft.AspNetCore.Mvc.Testing/Resources.resx

```diff
@@ -117,6 +117,9 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="InvalidAssemblyEntryPoint" xml:space="preserve">
+    <value>The provided class '{0}' in Assembly '{1}' is not an entry point to the assembly of the application. A common cause for this error is providing a class from a class library.</value>
```

Comment:
Type instead of class?

Also, the type itself doesn't need an entry point, it just needs to be in an assembly with an entry point.

Reply:
Thoughts on the changes?

Reply:
Type is better